corehq/apps/app_manager/tasks.py

```diff
@@ -44,12 +45,7 @@ def autogenerate_build(app, username):
 
 @serial_task('{app_id}-{version}', max_retries=0, timeout=60 * 60)
 def autogenerate_build_task(app_id, domain, version, comment):
-    app = get_app(domain, app_id)
-    errors = app.validate_app()
-    if not errors:
-        copy = app.make_build(comment=comment)
-        copy.is_auto_generated = True
-        copy.save(increment_version=False)
+    make_async_build_v2(app_id, domain, version, allow_prune=True, comment=comment)
```

Comment:
so `autogenerate_build_task` and `make_async_build_v2` are essentially identical?  Can we remove one of them?  (In a separate PR, perhaps)

Reply:
mostly likely - I didn't want to do that in this PR